packages/ava/src/text/tools/index.ts

```diff
@@ -0,0 +1,2 @@
+export * from './isSpecType';
+export { getVariableMap } from './getVariables';
```

Comment:
so what's the meaning of 'tools'? diff from 'utils'? 

This looks to me like an internal 'utils'.

Reply:
`utils` is for developer, similar to lodash.
`tools` will export for user.

Reply:
`tools` sounds like classes. any better name?

Reply:
`text/tools` rename to `text/utils`